index.ts

```diff
@@ -0,0 +1,217 @@
+import * as adapter from "@api3/airnode-adapter";
+import * as node from "@api3/airnode-node";
+import { AirnodeRrpFactory } from "@api3/airnode-protocol";
+import * as dotenv from "dotenv";
+import * as ethers from "ethers";
+import * as fs from "fs";
+import { each, isEmpty } from "lodash";
+import * as path from "path";
+import { safeDecode } from "./node/abi-encoding";
+//TODO: remove and use @api3/airnode-node import
+import { buildEVMProvider } from "./node/evm-provider";
+//TODO: remove and use @api3/airnode-node import
+import { removeKey, removeKeys } from "./node/object-utils";
+//TODO: remove and use @api3/airnode-node import
+import { getReservedParameters, RESERVED_PARAMETERS } from "./node/parameters";
+//TODO: remove and use @api3/airnode-node import
+import { deriveSponsorWallet } from "./node/wallet";
+//TODO: remove and use "@api3/airnode-protocol" import;
+import RrpBeaconServer from "./RrpBeaconServer.json";
```

Comment:
I guess you would import the contract and build it yourself

Reply:
I see that you're considering exporting the factory, that would also work (but it's a hassle to export factories for all the contracts we develop so we have to draw the line somewhere)

Reply:
Right, I was thinking about exporting the factory like with do for AirnodeRrp and a few others but I get your point.

Reply:
I think this was about TypeChain